config/environments/production.rb

```diff
@@ -79,43 +86,53 @@
   # Do not fallback to assets pipeline if a precompiled asset is missed.
   config.assets.compile = false
 
+  # Enable serving of images, stylesheets, and javascripts from an asset server.
+  # config.asset_host = "http://assets.example.com"
+
   # Generate digests for assets URLs
   config.assets.digest = true
 
   # Version of your assets, change this if you want to expire all your assets.
   config.assets.version = "1.0"
 
-  # Specifies the header that your server uses for sending files
+  # Specifies the header that your server uses for sending files.
   # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for apache
   config.action_dispatch.x_sendfile_header = "X-Accel-Redirect" # For nginx
   # If you have no front-end server that supports something like X-Sendfile,
   # just comment this out and Rails will serve the files
 
   # Force all access to the app over SSL, use Strict-Transport-Security,
   # and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
??? Seems like a big change.

Reply:
@pellaea: What do you think?

Reply:
@JoeCohen I don't believe this is much of a change. 

If I'm not mistaken we already default to SSL for all access. If you access the site via http://mushroomobserver.org, it responds with https://mushroomobserver.org as the address.

Also, I think [SSL is necessary to get HTTP/2](https://www.ssl.com/article/an-introduction-to-http2/). As of the Turbo PR, the app will be very slow without HTTP/2.

> Although the standard itself does not require the use of encryption, all major browser implementations (i.e. Firefox [09], Chrome, Safari, Opera, IE, Edge) have decided that they will only support HTTP/2 over TLS.
> 
> In fact, browsers distinguish between clear-text HTTP/2 and HTTP/2 over encrypted TLS as two different protocols. Encrypted HTTP/2 is called h2 and clear-text h2c. As of this writing, none of the major browsers support h2c , which means that TLS encryption is mandatory for a web site to take advantage of HTTP/2’s other advantages.

Reply:
@pellaea @mo-nathan, do you guys foresee problems forcing SSL, from the API or elsewhere? I feel like we _should_ be forcing SSL, but I don't know all our clients like you guys do.

I'd like to get moving on 7.1, either by trying out `config.force_ssl` in a small PR off `main`, or just merging this PR.

Reply:
We are now running the `config.force_ssl` since yesterday, without apparent problems. I may merge this tonight if i don't hear from anyone.